### DISCUSSION TITLE
重金求三角洲行动搭子带我飞

### DISCUSSION THREAD

**Discussion by HarmonyTrump** [share121/inter-knot#5472] [委托]
![img_localize_1dfea8c67b24fb6ea3001ba7d1ef5249_533x800](https://github.com/user-attachments/assets/2a0dd9c5-2e34-4654-a266-37816ee188dc)
有人一起玩三角洲行动吗？主要打大战场，不喜欢玩摸金，有人可以一起吗

**Comment by lbw2lmw** [comment]
👍

**Comment by Zhangyifan2008** [comment]
有，我叫义梵。Q区PC

**Comment by zzzplayer2010** [comment]
手游行吗

**Comment by QingMingqwq** [comment]
端游？

**Comment by jwhandwyn** [comment]
我我我，微信区名：但扒

**Comment by Yizakl** [comment]
山脚粥QV互通的
跟Co一样
